README.md

```diff
@@ -6,23 +6,94 @@ SPDX-License-Identifier: BSD-3-Clause
 
 # Energy balance and firn model (EBFM)
 
-This repository provides a Python implementation of a energy balance and firn model (EBFM).
+This repository provides a Python implementation of an energy balance and firn model (EBFM).
 
-Using YAC this model can be coupled to other models.
+Using YAC, this model can be coupled to other models.
 
 ## Preparations
 
-Please use the script `setup_venv.sh` to create a virtual environment for
-developing and running this dummy.
+### Create virtual environment
 
-## Running
+You can use an arbitrary location to create your virtual environment. 
+A common choice is `.venv`. In the following, we will use `$VENV` as a placeholder and the path to the virtual environment `.venv` should be set as follows:
+
+```sh
+export VENV=/path/to/your/environment/.venv
+```
+
+If you do not already have a virtual environment, you can create one using the following commands, which will additionally install the necessary dependencies specified in `pyproject.toml`:
+
+```sh
+python3 -m venv $VENV
+source $VENV/bin/activate
+pip install uv
+uv pip install -r pyproject.toml
```

Comment:
```suggestion
pip3 install "EBFM @ git+https://github.com/wardvp/EBFM.git"
```
I would rather install EBFM from git (or as soon as ready from PyPI) since this does not require the user to clone the repository.

For background: `uv` is just a helper that allows to extract the dependencies from `pyproject.toml` and install them. This is identical to `pip install -r requirements.txt`. However, if we install EBFM this will automatically take care of installing missing dependencies and there is no need for `uv`.

Also: If you are using `python3` I would also use `pip3` here.